blocks/footer/footer.css

```diff
@@ -1,14 +1,91 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
-}
-
 footer .footer {
-  max-width: 1200px;
+  max-width: 100vw;
   margin: auto;
+  background-color: #1a355e;
+  color: #fff;
+}
+
+.footer-main {
+  display: flex;
+  justify-content: space-between; 
+  flex-flow: row wrap; 
+  gap: 20px; 
+  padding-bottom: 64px;
+}
+
+.footer-column {
+  flex-grow: 1; 
+  min-width: 200px; 
+}
+
+.social-languages-container,
+.sublinks-default-container {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  flex-flow: row wrap;
+  margin-top: 20px;
+}
+
+.footer-social-wrapper,
+.footer-languages-wrapper,
+.footer-sublinks-wrapper,
+.default-content-wrapper {
+  flex: 1;
+  min-width: 200px;
+}
+
+.footer-social-row {
+  max-width: 100vw !important;
+  background-color: #65646a;
+  text-align: center;
+}
+
+.footer-sublinks-row {
+  max-width: 100vw !important;
+  background-color: white;
+  text-align: center;
+  font-family: 'Avenir Next Cyr W00 Regular', sans-serif;
 }
 
-footer .footer p {
+.footer-social-row h3,
+.footer-sublinks-row h3 {
+ margin-top: 0;
+}
+
+.footer-sublinks-row .default-content-wrapper {
+  color: #1a355e;
+}
+
+.footer-sublinks-row a {
+  color: #1a355e;
+  display: inline-block;
+  padding: 0 20px;
+}
+
+.footer-main h3 {
+  font-size: 17px;
```

Comment:
Better to use var(--heading-font-size-xs) or var(--body-font-size-xs) if possible.  This also allows better uniformity across breakpoints as we often adjust fonts up or down according to screen size.